src/content/docs/style-guide/writing-docs/article-templates/whats-new-tech-writer-tasks.mdx

```diff
@@ -15,28 +15,42 @@ A high-altitude view of the `What's new` process looks like this:
-* If a post is contributed by one of the reviewers listed in [CODEOWNERS](.github/CODEOWNERS), it doesn't need a product marketing review. You can start the writing review right away.
-* Review the doc using our main style guide rules. If you see major issues, work with the contributor (or the product marketing reviewer); otherwise, you can just make the changes yourself.
-* Confirm that the URLs to the docs site are absolute (relative paths don't resolve in the product UI):
+### The GitHub Board [#the-board]
+
+1. In the GitHub projects board, move the post from the triage column to the **Scheduled work** column. The post will remain there until the post is actually published.
+2. Insert the publishing date and time window into the PR title so it’s easy to recognize. Our [instructions](src/content/docs/style-guide/writing-docs/article-templates/whats-new-template.mdx) tell contributors that we offer these publishing windows: morning, mid-day, or evening. You may need to ask if they want this posted early to ensure it is available on time.
+
+### What to review [#the-review]
+
+1. Review the doc using our main style guide rules. If you see major issues, work with the contributor (or the product marketing reviewer); otherwise, you can just make the changes yourself.
+2. Confirm that the URLs to the docs site are absolute:
+  <Callout variant="important">
+  Relative paths don't resolve in the product UI which is why you need to use absolute paths for the posts.
+  </Callout>
   ```
     * Absolute path (Good): [**Billing user role**](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#roles)
     * Relative path (Bad): [**Billing user role**](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#roles)
   ```
-* If the post refers to any images, make sure they are not links to images hosted on other sites. Images files must reside in our repository.
-* Check if the post has the `isFeatured: true` in the front matter. If so, this means that the post will be pinned to the top of the product UI. You should remove the corresponding flag in the previously featured post so only one post is featured.
-* After you publish the post, confirm that it shows up in both the docs site and in the product UI.
-* For posts that are to be published in the future:
-  * Generally, the hero on duty takes responsibility for the post on the day it needs to be published. 
+3. If the post refers to any images, make sure they are not links to images hosted on other sites. Images files must reside in our repository.
+4. Check if the post has the `isFeatured: true` in the front matter. If so, this means that the post will be pinned to the top of the product UI. You should remove the corresponding flag in the previously featured post so only one post is featured.
+
+### Publish the post
```

Comment:
Anchor

Reply:
@urbiz-nr, I"m not sure what "Anchor" means in this context. Were you suggesting I add one? If so, where?

Reply:
In the header! As in [#publish-post] @rhetoric101.

Reply:
@urbiz-nr, I finally see you were pointing out a missing anchor in the header! I'll fix that! Thanks!